docker/requirements.txt

```diff
@@ -5,10 +5,10 @@
 empy
 GitPython
 pyyaml
+ros_buildfarm
 
 ###### Requirements with Repo Specifiers ######
 #   See https://pip.readthedocs.io/en/stable/reference/pip_install/#git
-git+https://github.com/ros-infrastructure/ros_buildfarm.git@master#egg=ros_buildfarm
```

Comment:
@ruffsl I believe there is everything we need in the released ros_buildfarm package so we dont need to install from github anymore. Does that seem ok to you?

Reply:
Sounds good!